pkg/kubernetes/scripts/policy.js

```diff
@@ -236,7 +287,6 @@
                 var name = toName(role);
                 var binding = {
                     kind: "RoleBinding",
-                    apiVersion: "v1",
```

Comment:
Is it safe to drop this for older OpenShift releases too? We only test with 3.6, but AFAIR this was supposed to be compatible down to 3.4?

Reply:
Yes, it's been v1 for a long time now.